java/arcs/android/sdk/host/AndroidHost.kt

```diff
@@ -31,16 +31,24 @@ import kotlinx.coroutines.runBlocking
 abstract class AndroidHost(
     val context: Context,
     lifecycle: Lifecycle,
-    val activationFactory: ActivationFactory = ServiceStoreFactory(context),
+    coroutineContext: CoroutineContext,
+    arcSerializationContext: CoroutineContext,
     schedulerProvider: SchedulerProvider,
     vararg particles: ParticleRegistration
-) : JvmHost(schedulerProvider, *particles), DefaultLifecycleObserver {
-
+) : AbstractArcHost(
+    coroutineContext = coroutineContext,
+    updateArcHostContextCoroutineContext = arcSerializationContext,
+    schedulerProvider = schedulerProvider,
+    initialParticles = *particles
+), DefaultLifecycleObserver {
     init {
         lifecycle.addObserver(this)
     }
 
-    override val stores: StoreManager = StoreManager(activationFactory)
+    override val platformTime = JvmTime
+
+    @ExperimentalCoroutinesApi
+    override val stores: StoreManager = StoreManager()
```

Comment:
This doesn't need the activation factory anymore?

Reply:
Should still be there, fixed.